Lib/vanilla/vanillaLevelIndicator.py

```diff
@@ -3,7 +3,7 @@
 
 # This control is available in OS 10.4+.
 # Cause a NameError if in an earlier OS.
-NSLevelIndicator
+# NSLevelIndicator
```

Comment:
Read the comment above this line :)

Reply:
On the other hand, I doubt we meaningfully support 10.4, so maybe that entire section should go.

Reply:
Now I see.